backend/app/api/endpoints/users.py

```
@@ -251,6 +263,25 @@ async def read_wegent_runtime_user(
     )
 
 
+@router.post("/me/wegent-runtime-token", response_model=WegentRuntimeAuthTokenResponse)
+async def create_wegent_runtime_auth_token(
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)